src/OneExplorer/OneExplorer.ts

```diff
@@ -165,6 +165,10 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<OneNode> {
 
   // TODO: Add move()
 
+  openContainingFolder(oneNode: OneNode): void {
+    vscode.commands.executeCommand('revealFileInOS', oneNode.node.uri);
+  }
+
```

Comment:
Honestly, I expected the `open containing folder` button will show the folder in file explorer view, which can be executed by ` vscode.commands.executeCommand('revealInExplorer', oneNode.node.uri);`.
I'm really sorry if it has been discussed before, but can I get the background of opening the os' file explorer?

Reply:
I think those commands means to do the same, based on their name..? Where is the command id from? I tried to grep one from vscode api docs, and I cannot find one. 

I can find the revealFileInOs [here](https://code.visualstudio.com/updates/v1_10)

Reply:
After our offline talk, I got the history about this feature (popping up OS File Window). It's requested in our discussion, and the feature what I expected is already available by clicking *file explorer icon*. :smile: